genesyscloud/util/resourcedata/resourcedata.go

```diff
@@ -14,12 +15,70 @@ const (
 	DateParseFormat = "2006-01-02"
 )
 
+// Use these functions to read properties from the schema and set it on in a map in build function
+
+// This function will read a map and set a property on an object if the value exists
+func BuildSDKStringValueIfNotNil(field **string, targetMap map[string]interface{}, key string){
```

Comment:
Is this correct I have never a double pointer to a string

Reply:
It is, I don't know why but it wasn't setting the value when I was trying with one but I could be wrong